packages/dev/src/main.ts

```diff
@@ -121,7 +123,7 @@ interface HandleOptions {
   headersCollector?: HeadersCollector
 }
 
-export type ResponseType = 'edge-function' | 'function' | 'redirect' | 'static'
+export type ResponseType = 'edge-function' | 'function' | 'image' | 'redirect' | 'static'
```

Comment:
Should this be considered a breaking change? There is new possible response type for public `handleAndIntrospect(NodeRequest)` methods

Reply:
Not sure that's worth a major bump, but happy to defer to you.

Reply:
I don't think we should consider widening a type as a breaking change